scripts/tns_run.py

```diff
@@ -0,0 +1,20 @@
+import os
+from stream2hop import Utils as ut
+import json
+
+region = "us-west-2"
+cred_secret_name = "dev-gcn2hop-hopcreds"
+tns_api_key_secret_name = "tns_api_key"
+configDir = "/root/share"
+Location = "%s/kafkacat.conf" % configDir
+
+
+os.system("mkdir -p %s" % configDir)
```

Comment:
```suggestion
os.system(f"mkdir -p {configDir}")
```
Prefer f-strings, which are more readable.

Reply:
will do